app/views/layouts/application.html.erb

```diff
@@ -9,8 +9,12 @@
     <%= stylesheet_link_tag 'application', media: 'all' %>
   </head>
   <body>
-    <p class="notice"><%= notice %></p>
-    <p class="alert"><%= alert %></p>
+    <% if notice %>
```

Comment:
Sorry for the overlap--I did this over in https://github.com/cheshire137/overwatch-team-comps/pull/42/files#diff-9599427925097c3c66f26ac1e0de5cadL12 too because the extra space at the top of the page bothered me!